xain_fl/coordinator/coordinator.py

```diff
@@ -304,8 +311,9 @@ def _handle_heartbeat(
         logger.debug(
             "Heartbeat response",
             participant_id=participant_id,
-            message=state,
+            state=_STATE.values_by_number[state].name,
```

Comment:
It might be worth adding a little helper if we're doing that in other places (I see we're doing this above already):

```python
def pb_enum_to_str(pb_enum, variant):
    return pb_enum.values_by_number[variant].name
```
Then the above could be a bit more digest:

```
state=pb_enum_to_str(_STATE, state)
```

Reply:
Good point👍 I have added the helper function.